topo/node/srl/testdata/configpush_failure

```diff
@@ -10,13 +10,21 @@ A:pod1# environment complete-on-space false
 A:pod1# info from state system app-management application mgmt_server state | grep running
                 state running
 --{ running }--[  ]--
-A:pod1# info from state system configuration commit 1 status | grep complete
-                status complete
+A:pod1# file cat /etc/opt/srlinux/devices/app_ephemeral.mgmt_server.ready_for_config
+loaded initial configuration
```

Comment:
in srlinux-scrapli package v0.6.0 we improved the way to check when the NOS is ready to accept configs. This is the new way